ESP8266/ESP8266.cpp

```diff
@@ -286,7 +286,7 @@ bool ESP8266::send(int id, const void *data, uint32_t amount)
         if (_parser.send("AT+CIPSEND=%d,%lu", id, amount)
             && _parser.recv(">")
             && _parser.write((char*)data, (int)amount) >= 0) {
-            _parser.process_oob(); // multiple sends in a row require this
+            while (_parser.process_oob()); // multiple sends in a row require this
```

Comment:
How long can this block?

Reply:
It might block for half a second

Reply:
Same ~0.5 second delay occurs also with ethernet every now and then